### ISSUE TITLE
Curve Labs

### ISSUE THREAD

**Issue by cryptodani** [aragon/nest#194]
# Curve Labs

<br>

## About

**Project name**: Curve Labs
<br>
**Project description:** A dedicated team is needed for research, design, and user on-boarding for Continuous Organizations on the Aragon Network with the Fundraising app and it's derivatives.

**Link to our full Nest proposal**: https://hackmd.io/NgzBo3CSROidSOa6aUq9Jg

**TL;DR:**
**Product: Aragon Fundraising app**

- bonding curves unlock a financial paradigm shift
- Aragon DAOs allow project governance & capital management post-fundraising
- significant interest from early adopters

**Product Market Fit: Cultivate best practices & iterate on user feedback**

- talk to users to identify pain points and improve UI/UX
- identify legal frameworks and jurisdictions that are favorable for autonomous organizations
- pilot program with high impact projects to build user profiles and lay the foundation for more users in the same target market

**Growth: next-gen DeFi platform on Aragon**

- widen the total addressable market through tested templates, compliant PoCs and DAICO business models
- experiment with novel uses of curve bonding as cryptoeconomic primitive and establish valuation models around its implementation
- establishing communication channels to engage target markets
- hosting economic productivity in Aragon orgs & facilitating value flow to ANT

**Code repo and/or website URL (if any):** https://github.com/AragonBlack/fundraising

**Where did you hear about the Nest program and what prompted you to apply?**
Long standing presence in the Aragon ecosystem.

<br>

## Team

**Name of the project lead:** 
**Cem Dagdelen :** Project strategy lead, user outreach and support, on-boarding design and improvement.

**Email address of the project lead (whoever is filling out this application):** cem@aragon.black

**List each other member that is/would be working on this project (LinkedIn/GitHub profiles are preferred but not required)
What are the main roles of each team member?**

**Daniel Shavit :** Mechanism design, curve configuration, valuation research and product development.

**Jack Laing:** Publishing documentation on curve designs and configurations, onboarding user guides, and AF DAICO best practices.

**Has the team already worked together? (please provide details)**
Yes, Cem and Dani have worked for 5+ years together, on some crypto startups and DAO protocols.
Jack Laing has previously worked at Colony and founded OrgTech, he has collaborated with Cem and Dani within the scope of DAOIncubator.

**Is your team already present on the Aragon Forum? If Yes, what are your usernames?**
@cemfd
@daniels
<br>

## Progress

**How far along are you?**
We've deployed a Rinkeby version with mainnet coming in the next month.

**Is anyone using your product/application?**
**If yes, How many active users do you have and what kind of feedback have you received?**
Yes, 133 Rinkeby DAOs deployed to date.
Feedback points to more support needed, ux improvements and designs/integrations for compliance.

**If you've applied previously with the same idea, how much progress have you made since the last time you applied? What has changed?**
No previous similar proposal.

<br>

## Idea Problem

**Why did you pick this idea to work on? Do you have domain expertise in this area?**
We picked this idea due to the concomitant timing of:
- massive demand for bonding curve applications and fairer and more transparent modes of decentralized fundraising applications
-the release of the Fundraising app
- feedback from a continuous work of onboarding and validation 
We have been part of the DAO space since 2 years and have been performing R&D, development and outreach for the Fundraising app for 1year+.

**What's new about what you're making? What do people resort to because it doesn't exist yet (or they don't know about it)?**
Aragon Fundraising provides the 1st mainnet implementation of the DAICO model.
- first mover with regards to audited bonding curves
- ties in to Aragon governance capabilities to manage capital
- front-running resistant with batched orders
- supports multiple collaterals
- customizable collateral weight according to use-case

**Have you talked to people about what you're making? Do people want it and do you have examples of this?**

Curve Labs has talked with 60+ projects. Of these, the most promising initial target markets seem to be:
-impact investment
-DeFi protocols
-data marketplaces

<br>

## Market

**What is the target market or end user in mind for this project?**
**Initial Target Market:** high-leverage partners/pilots that are representative of a target market. Once successfully completed, these pilot programs will enable similar projects and users to follow the same template.

Potential markets:
-Open source projects
-Impact bonds
-Marketplaces
-Political organizations
-Grant communities
-Self-organizing communities + spaces

**What are the alternatives or competitors to this project?**
DAOstack/Gnosis, Fairmint, and Commons Stack all have similarities. It is yet to be determined whether or not these projects will be complementary or adversarial to Aragon Fundraising. In the case of Commons Stack it is viewed as a partner project due to ideological alignment and potential usage of the Aragon stack.

**How will this project get users? If your project is the type that faces a chicken-and-egg problem in the sense that it won't be attractive to users until it has a lot of users (e.g. a marketplace), how will you overcome that?**
The Fundraising app is partly aimed at resolving these issues for those assets and marketplace by providing a decentralized market maker with pricing at any level of liquidity.

<br>

## Legal

**Will you be using an Aragon DAO for operating this project and receiving the funds?** Yes/No
Yes
**Are any of the project team members covered by non-competes or intellectual property agreements that overlap with your project? If so, please explain.**
No
<br>

## Aragon

Why do you think Aragon is a good fit for this project?
We are long standing members of the Aragon community and have been involved in nest and flock teams as well as the Events and Nest DAO. We believe we have strong strategical and cultural alignement with the network.

Are there any existing Aragon projects that are similar or complementary to your project?
Conviction Voting is a promising complementary app that enhances Aragon Fundraising with a novel voting scheme. Governance features complete the value proposition towards a holistic DAICO product. More details on a potential collaboration between Aragon Fundraising and Conviction Voting can be found here. 

<br>

## Funding

**How much money are you requesting as a grant? (the maximum grant size we provide is $150k in USD value)**
USD 150k+30k ANT

**Have you already received any grants or funds from 3rd parties? If yes, how much?**
We've been part of teams recipients of nest and flock grants.

Are you applying for funding to any other DAO platforms or grant programs? If so, that is not a problem as it increases the likelihood of your success, but it's important that we can take that into consideration if your project relies on multiple grants.
We haven't 

**How do you plan to deploy grant funds? (split between research/development/marketing/etc):**
40% R&D and development
40% Marketing + Network building + Workshops + Publishing
20% Legal R&D

**What is your expected monthly burn rate?**

25k USD.

Do you think this project will require additional funds once you have depleted the grant funds? If so, why? Where do you plan on acquiring further funding?

<br>

## Roadmap

**How long do you plan on working on this project before shipping?**
We have been working on this project for over a year and are shipping a main-net app in the near future. 
What are the key milestones you have to accomplish before you can ship a production ready product to users?
We are hoping to achieve before the next iteration of our first production ready shipment:
- ux and onboarding improvements
- onboarded pilot projects and design publications that can provide inspiration 
- a path to compliance in select jurisdictions

<br>

## Notes

Is there anything else you would like to share or ask us?

<br>
<hr>
<br>

Thank you for applying! A Nest team member will read and reply to this application within 72 hours. They will ask questions to better understand your project and then they will help you improve your application to maximize your success in the Nest program. If you have any questions as to what this looks like or next steps, please see our [Frequently Asked Questions](https://github.com/temp-nestdao/nest/blob/master/FAQ.md) page for more info.

Once again, thank you for applying to the Nest program. We look forward to working with you!

**Comment by burrrata** [comment]
Thanks for the application! We're already in contact working on this, but adding a message here for historical purposes :)

**Comment by burrrata** [comment]
While I really really really like this proposal, it's currently not a good fit for the Nest program. This is because of the high volume of research and the fact that so many Nest members are part of it. It's with great regret that I must close this Issue, however, if anyone wants to modify and reopen the Issue for further consideration that would be warmly welcomed :)

**Comment by pythonpete32** [comment]
I will be working on this issue with the curve labs team

**Comment by 0x6431346e** [comment]
> While I really really really like this proposal, it's currently not a good fit for the Nest program.

Then what is this proposal a good fit for?
Because it looks like is [not a good fit for an AGP either](https://forum.aragon.org/t/simplifying-anv-5-ensuring-the-long-term-sustainability-of-the-aragon-network/1514).

AA blocked the AGP suggesting this team should apply to Nest, then Nest says they're not "a good fit" :roll_eyes:. As far as I know there are no other grant programs which can fit this budget.

Will AA consider funding this using their discretionary budget or will this get pushed to ANV-6, essentially setting them back by 3 months, a period in which they're not even getting proper feedback, other than, "risky", "high-volume of research".

> fact that so many Nest members are part of it.

The conflict of interest argument is bogus and a counter argument of censorship can be made considering you're part of both AA and Nest. As far as I remember you are also a part of 1hive, a team that has been granted funds by Nest.

Did this at least go through a vote within the Nest DAO? Where can I read about the reasoning behind this decision?

The bottom line is that the push back from the AA and Nest is **very discouraging to applicants** when there is **no proper feedback** given. A grant system is as good as the feedback it provides IMO.
